java/ql/test/query-tests/security/CWE-200/semmle/tests/options

```diff
@@ -1 +1 @@
-//semmle-extractor-options: --javac-args -cp ${testdir}/../../../../../stubs/apache-commons-lang3-3.7/
\ No newline at end of file
+//semmle-extractor-options: --javac-args -cp ${testdir}/../../../../../stubs/apache-commons-lang3-3.7/:${testdir}/../../../../../stubs/google-android-9.0.0
```

Comment:
Is the `apache-commons-lang` stub needed?

Reply:
Yes, but not for these tests. That's for a pre-existing test for another CWE-200 test: `TempDirLocalInformationDisclosure.qlref`

Reply:
Ah makes sense, thanks!